sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModel.cs

```diff
@@ -21,5 +21,13 @@ public CommunicationIdentifierModel(string rawId, CommunicationUserIdentifierMod
         {
             Kind = kind;
         }
+
+        public CommunicationIdentifierModel(string rawId, CommunicationUserIdentifierModel communicationUser, PhoneNumberIdentifierModel phoneNumber, MicrosoftTeamsUserIdentifierModel microsoftTeamsUser)
```

Comment:
I'm not quite sure where this is coming from. @LuChen-Microsoft ?

Reply:
I would ignore it. This comes from retention policy changes

Reply:
I modified existing code to accommodate the new generation changes. I believe the generation changes are intended.

Reply:
@LuChen-Microsoft does this change make sense?

Reply:
This comes from Dominik's change for CommunicationIdentifier